src/cobra/util/util.py

```diff
@@ -1,27 +1,62 @@
-"""General utilities."""
+"""General utilities used across the package."""
+
+from typing import Any
 
 from depinfo import print_dependencies
 
 
-def format_long_string(string, max_length=50):
+def format_long_string(string: str, max_length: int = 50) -> str:
+    """Shorten long string into a small string with ellipsis.
+
+    Parameters
+    ----------
+    string: str
+        The long string to shorten.
+    max_length: int, optional
+        The maximum length after which to append ellipsis (default 50).
+
+    Returns
+    -------
+    str
+        The shortened string.
+
+    """
     if len(string) > max_length:
-        string = string[: max_length - 3]
-        string += "..."
+        string = f"{string[: max_length - 3]}..."
     return string
```

Comment:
Can you replace the body of the function with [textwrap.shorten](https://docs.python.org/3/library/textwrap.html#textwrap.shorten), please?

```py
return textwrap.shorten(string, width=max_length, placeholder="...")
```

Reply:
A future refactoring could then be to replace all calls to `format_long_string` with `textwrap.shorten` directly.

Reply:
Perfect.